src/c4/ifpstream.cc

```diff
@@ -0,0 +1,69 @@
+//----------------------------------*-C++-*-----------------------------------//
+/*!
+ * \file   c4/ifpstream.cc
+ * \author Mathew Cleveland
+ * \date   Feb. 2020
+ * \brief  Define methods of class ifpstream
+ * \note   Copyright (C) 2018-2020 Triad National Security, LLC.
```

Comment:
Please fix dates.

Reply:
Do you mean the copyright date or the the main date, or both?

Reply:
Dates look okay now.  Maybe I reviewed an earlier revision.

Reply:
Edit: Actually the copyright date should just be 2020.  Not '2018-2020'.

Reply:
Fixed.